### DISCUSSION TITLE
Disable code viewer

### DISCUSSION THREAD

**Discussion by ghost** [doublecmd/doublecmd#1472] [Suggestions]
Please make it possible to disable the code view mode. Or at least give an opportunity to adjust fonts and colours of code highlighting. Right now it looks terrible (too bold):
![зображення](https://github.com/doublecmd/doublecmd/assets/28775275/ada2b290-eacd-4485-ad45-a21845712149)
I have to keep pressing "1".

**Comment by JonnyTech** [comment]
Font is selectable: Configuration >> Options >> Fonts >> Viewer Font

Or you can select another viewer: Configuration >> Options >> Tools >>  Viewer >> Use external program

**Comment by alexx2000** [comment]
You can disable it, see [SynEditMask](https://skif-off.github.io/doublecmd.github.io/doc/en/configxml.html) option.

**Comment by ghost** [comment]
> You can disable it, see [SynEditMask](https://skif-off.github.io/doublecmd.github.io/doc/en/configxml.html) option.

Yeah, it works. Thank you!
